src/main/kotlin/app/ui/profilePreview/View/ProfilePreview.kt

```diff
@@ -171,16 +164,11 @@ class ProfilePreview: View() {
                                 fontSize = 2.em
                                 textFill = c("#CC4141")
                             }
-
-                            action {
-
-                            }
+                            // TODO("insert action here when user clicks next button")
                         }
-
```

Comment:
I went in an removed a lot of the whitespace in this files. I feel how you see it all here doesn't really give an accurate representation of it though